checkLogos--backup.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const tokensDirectory = './src/tokens';
+const request = require('request');
+
+function checkTokenLogos() {
+  let invalidImgArray = [];
+  fs.readdirSync(tokensDirectory).forEach(folder => {
+    fs.readdirSync(`${tokensDirectory}/${folder}`).forEach(file => {
+      const fullPath = `${tokensDirectory}/${folder}/${file}`;
+      const obj = JSON.parse(fs.readFileSync(fullPath, 'utf8'));
+
+      if (obj.logo.src === '') {
+        return;
+      }
+
+      request({ method: 'HEAD', uri: obj.logo.src }, function(err, response) {
+        if (err) {
+          invalidImgArray.push(obj.logo.src);
+        }
+
+        if (!err && response.statusCode === 400) {
+          invalidImgArray.push(obj.logo.src);
```

Comment:
Do we need both conditionals for error 400 and not 400?  Might be able to just use line 17s conditional.

Reply:
Oh, just noticed this is just to preserve the old checkLogos file.  In that case, do we need the backup?  Can probably just delete it.